depends/config.site.in

```diff
@@ -0,0 +1,100 @@
+cross_compiling=maybe
+host_alias=@HOST@
+ac_tool_prefix=${host_alias}-
+
+if test -z $with_boost; then
+  with_boost=$prefix
+fi
+if test -z $with_qt_plugindir; then
+  with_qt_plugindir=$prefix/plugins
+fi
+if test -z $with_qt_translationdir; then
+  with_qt_translationdir=$prefix/translations
+fi
+if test -z $with_qt_bindir; then
+  with_qt_bindir=$prefix/native/bin
+fi
+if test -z $with_protoc_bindir; then
+  with_protoc_bindir=$prefix/native/bin
+fi
+if test -z $with_comparison_tool; then
+  with_comparison_tool=$prefix/native/share/BitcoindComparisonTool_jar/BitcoindComparisonTool.jar
+fi
+
+
+if test -z $enable_wallet && test -n "@no_wallet@"; then
+  enable_wallet=no
+fi
+
+if test -z $with_miniupnpc && test -n "@no_upnp@"; then
+  with_miniupnpc=no
+fi
+
+if test -z $with_gui && test -n "@no_qt@"; then
```

Comment:
If you're getting rid of these, you may as well get rid of the entire config.site.in generation. You'll want to replace it with something though, so that your build-system can reference the deps config.